solutions/USACO jan21-uddered-bronze.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: usaco-1083
+source: USACO Bronze 2021 January
+title: Uddered but not Herd
+author: Ananth Kashyap
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob3_bronze_jan21.html)
```

Comment:
this is the wrong link

Reply:
also, the official analysis already has a python solution (although it's very short, so it's probably fine to have a longer, more clear implementation)